src/arrayMethodJoin.js

```diff
@@ -5,7 +5,23 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let str = '';
+    let sep = '';
+    separator === undefined
+      ? (sep = ',')
```

Comment:
before writing condition you have to write the element you're working with
`sep = ...`

Reply:
you still have nested ternary operators